libbpf-tools/capable.bpf.c

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Unique filtering based on
+// https://github.com/libbpf/libbpf-rs/tree/master/examples/capable
+//
+// Copyright 2022 Sony Group Corporation
+
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include "capable.h"
+
+#define MAX_ENTRIES	10240
+
+extern int LINUX_KERNEL_VERSION __kconfig;
+
+const volatile pid_t bpf_pid = -1;
```

Comment:
`bpf_pid` ? Could this be renamed to something like `my_pid` ?

Reply:
Renamed `bpf_pid` -> `my_pid`